packages/sankey/src/types.ts

```diff
@@ -162,6 +168,7 @@ export interface SankeyCommonProps<N extends DefaultNode, L extends DefaultLink>
     labelPadding: number
     labelOrientation: 'horizontal' | 'vertical'
     labelTextColor: InheritedColorConfig<SankeyNodeDatum<N, L>>
+    labelComponent: ComponentType<SankeyLabelProps>
```

Comment:
I usually only support a `FunctionComponent`, I don't think we should allow class components.

Reply:
Fixed. thanks